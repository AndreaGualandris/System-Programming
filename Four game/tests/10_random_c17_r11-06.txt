8
8
6
11
6
2
12
1
0
2
1
3
8
6
13
16
8
13
12
1
15
2
4
5
4
4
3
6
1
15
6
11
11
6
3
9
8
16
6
6
3
14
8
8
0
1
0
11
6
11
15
0
8
15
11
14
13
13
13
16
4
10
9
2
2
8
1
1
9
0
1
8
14
10
10
9
16
0
10
15
0
2
13
11
4
7
9
1
9
16
2
3
8
14
6
8
3
5
1
7
7
5
15
0
12
3
13
2
14
16
9
0
6
9
11
5
4
14
2
1
12
11
5
10
12
11
0
6
7
4
14
9
12
12
12
0
14
6
6
4
11
9
1
5
13
13
7
5
14
5
12
8
3
1
8
16
8
1
8
10
11
12
14
3
3
11
16
5
11
0
6
6
10
4
13
5
9
16
7
4
1
8
2
2
10
5
11
