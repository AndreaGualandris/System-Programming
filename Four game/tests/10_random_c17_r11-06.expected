-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
-----------------
01234567890123456
Player 'o' to move:
        o
        v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
        o        
-----------------
01234567890123456
Player 'x' to move:
        x
        v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
        x        
        o        
-----------------
01234567890123456
Player 'o' to move:
      o
      v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
        x        
      o o        
-----------------
01234567890123456
Player 'x' to move:
           x
           v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
        x        
      o o  x     
-----------------
01234567890123456
Player 'o' to move:
      o
      v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
      o x        
      o o  x     
-----------------
01234567890123456
Player 'x' to move:
  x
  v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
      o x        
  x   o o  x     
-----------------
01234567890123456
Player 'o' to move:
            o
            v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
      o x        
  x   o o  xo    
-----------------
01234567890123456
Player 'x' to move:
 x
 v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
      o x        
 xx   o o  xo    
-----------------
01234567890123456
Player 'o' to move:
o
v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
      o x        
oxx   o o  xo    
-----------------
01234567890123456
Player 'x' to move:
  x
  v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
  x   o x        
oxx   o o  xo    
-----------------
01234567890123456
Player 'o' to move:
 o
 v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
 ox   o x        
oxx   o o  xo    
-----------------
01234567890123456
Player 'x' to move:
   x
   v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
 ox   o x        
oxxx  o o  xo    
-----------------
01234567890123456
Player 'o' to move:
        o
        v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
        o        
 ox   o x        
oxxx  o o  xo    
-----------------
01234567890123456
Player 'x' to move:
      x
      v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
      x o        
 ox   o x        
oxxx  o o  xo    
-----------------
01234567890123456
Player 'o' to move:
             o
             v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
      x o        
 ox   o x        
oxxx  o o  xoo   
-----------------
01234567890123456
Player 'x' to move:
                x
                v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
      x o        
 ox   o x        
oxxx  o o  xoo  x
-----------------
01234567890123456
Player 'o' to move:
        o
        v
-----------------
                 
                 
                 
                 
                 
                 
                 
        o        
      x o        
 ox   o x        
oxxx  o o  xoo  x
-----------------
01234567890123456
Player 'x' to move:
             x
             v
-----------------
                 
                 
                 
                 
                 
                 
                 
        o        
      x o        
 ox   o x    x   
oxxx  o o  xoo  x
-----------------
01234567890123456
Player 'o' to move:
            o
            v
-----------------
                 
                 
                 
                 
                 
                 
                 
        o        
      x o        
 ox   o x   ox   
oxxx  o o  xoo  x
-----------------
01234567890123456
Player 'x' to move:
 x
 v
-----------------
                 
                 
                 
                 
                 
                 
                 
        o        
 x    x o        
 ox   o x   ox   
oxxx  o o  xoo  x
-----------------
01234567890123456
Player 'o' to move:
               o
               v
-----------------
                 
                 
                 
                 
                 
                 
                 
        o        
 x    x o        
 ox   o x   ox   
oxxx  o o  xoo ox
-----------------
01234567890123456
Player 'x' to move:
  x
  v
-----------------
                 
                 
                 
                 
                 
                 
                 
        o        
 xx   x o        
 ox   o x   ox   
oxxx  o o  xoo ox
-----------------
01234567890123456
Player 'o' to move:
    o
    v
-----------------
                 
                 
                 
                 
                 
                 
                 
        o        
 xx   x o        
 ox   o x   ox   
oxxxo o o  xoo ox
-----------------
01234567890123456
Player 'x' to move:
     x
     v
-----------------
                 
                 
                 
                 
                 
                 
                 
        o        
 xx   x o        
 ox   o x   ox   
oxxxoxo o  xoo ox
-----------------
01234567890123456
Player 'o' to move:
    o
    v
-----------------
                 
                 
                 
                 
                 
                 
                 
        o        
 xx   x o        
 ox o o x   ox   
oxxxoxo o  xoo ox
-----------------
01234567890123456
Player 'x' to move:
    x
    v
-----------------
                 
                 
                 
                 
                 
                 
                 
        o        
 xx x x o        
 ox o o x   ox   
oxxxoxo o  xoo ox
-----------------
01234567890123456
Player 'o' to move:
   o
   v
-----------------
                 
                 
                 
                 
                 
                 
                 
        o        
 xx x x o        
 oxoo o x   ox   
oxxxoxo o  xoo ox
-----------------
01234567890123456
Player 'x' to move:
      x
      v
-----------------
                 
                 
                 
                 
                 
                 
                 
      x o        
 xx x x o        
 oxoo o x   ox   
oxxxoxo o  xoo ox
-----------------
01234567890123456
Player 'o' to move:
 o
 v
-----------------
                 
                 
                 
                 
                 
                 
                 
 o    x o        
 xx x x o        
 oxoo o x   ox   
oxxxoxo o  xoo ox
-----------------
01234567890123456
Player 'x' to move:
               x
               v
-----------------
                 
                 
                 
                 
                 
                 
                 
 o    x o        
 xx x x o        
 oxoo o x   ox x 
oxxxoxo o  xoo ox
-----------------
01234567890123456
Player 'o' to move:
      o
      v
-----------------
                 
                 
                 
                 
                 
                 
      o          
 o    x o        
 xx x x o        
 oxoo o x   ox x 
oxxxoxo o  xoo ox
-----------------
01234567890123456
Player 'x' to move:
           x
           v
-----------------
                 
                 
                 
                 
                 
                 
      o          
 o    x o        
 xx x x o        
 oxoo o x  xox x 
oxxxoxo o  xoo ox
-----------------
01234567890123456
Player 'o' to move:
           o
           v
-----------------
                 
                 
                 
                 
                 
                 
      o          
 o    x o        
 xx x x o  o     
 oxoo o x  xox x 
oxxxoxo o  xoo ox
-----------------
01234567890123456
Player 'x' to move:
      x
      v
-----------------
                 
                 
                 
                 
                 
      x          
      o          
 o    x o        
 xx x x o  o     
 oxoo o x  xox x 
oxxxoxo o  xoo ox
-----------------
01234567890123456
Player 'o' to move:
   o
   v
-----------------
                 
                 
                 
                 
                 
      x          
      o          
 o    x o        
 xxox x o  o     
 oxoo o x  xox x 
oxxxoxo o  xoo ox
-----------------
01234567890123456
Player 'x' to move:
         x
         v
-----------------
                 
                 
                 
                 
                 
      x          
      o          
 o    x o        
 xxox x o  o     
 oxoo o x  xox x 
oxxxoxo ox xoo ox
-----------------
01234567890123456
Player 'o' to move:
        o
        v
-----------------
                 
                 
                 
                 
                 
      x          
      o o        
 o    x o        
 xxox x o  o     
 oxoo o x  xox x 
oxxxoxo ox xoo ox
-----------------
01234567890123456
Player 'x' to move:
                x
                v
-----------------
                 
                 
                 
                 
                 
      x          
      o o        
 o    x o        
 xxox x o  o     
 oxoo o x  xox xx
oxxxoxo ox xoo ox
-----------------
01234567890123456
Player 'o' to move:
      o
      v
-----------------
                 
                 
                 
                 
      o          
      x          
      o o        
 o    x o        
 xxox x o  o     
 oxoo o x  xox xx
oxxxoxo ox xoo ox
-----------------
01234567890123456
Player 'x' to move:
      x
      v
-----------------
                 
                 
                 
      x          
      o          
      x          
      o o        
 o    x o        
 xxox x o  o     
 oxoo o x  xox xx
oxxxoxo ox xoo ox
-----------------
01234567890123456
Player 'o' to move:
   o
   v
-----------------
                 
                 
                 
      x          
      o          
      x          
      o o        
 o o  x o        
 xxox x o  o     
 oxoo o x  xox xx
oxxxoxo ox xoo ox
-----------------
01234567890123456
Player 'x' to move:
              x
              v
-----------------
                 
                 
                 
      x          
      o          
      x          
      o o        
 o o  x o        
 xxox x o  o     
 oxoo o x  xox xx
oxxxoxo ox xooxox
-----------------
01234567890123456
Player 'o' to move:
        o
        v
-----------------
                 
                 
                 
      x          
      o          
      x o        
      o o        
 o o  x o        
 xxox x o  o     
 oxoo o x  xox xx
oxxxoxo ox xooxox
-----------------
01234567890123456
Player 'o' wins!
